### PR TITLE
Add Bengali translation and fix missing keys bug

### PR DESCRIPTION
**Pull request by alicoding**
If we include a partially translated language it will cause React-Intl
to throw because it can't find a key name in the object that we pass to
intlData object. I'm using Object.assign (shimmed) to always merge
missing keys based on en-US one.

### CODE REVIEW

**Comment by alicoding** [comment]
@k88hudson R?

File: src/lib/render.jsx [hunk 1]
```diff
@@ -1,4 +1,5 @@
 var React = require('react');
+var ObjectAssign = require('object-assign');
```
Comment (k88hudson) [inline src/lib/render.jsx]: we've been using `var assign = require('react/lib/Object.assign')`

File: src/locales/bn-BD.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+bn-BD: {username: ইউজারনেম, password: পাসওয়ার্ড, email: ইমেইল, email_me_updates: ওয়েবমেকার
```
Comment (k88hudson) [inline src/locales/bn-BD.yaml]: is it just me or is the spacing weird here

**Comment by k88hudson** [comment]
@alicoding small nit but this looks good

**Comment by alicoding** [comment]
@k88hudson R? :)

File: package.json [hunk 3]
```diff
@@ -68,6 +68,7 @@
     "color": "0.8.0",
     "lodash.defaults": "3.1.1",
     "lru-cache": "2.6.5",
+    "object-assign": "^3.0.0",
```
Comment (k88hudson) [inline package.json]: remove this

**Comment by k88hudson** [comment]
@alicoding just remove that dep and you're good

**Pull request merged**